common/util.go

```diff
@@ -227,7 +236,11 @@ func ProfileFormatMimeType(f ffmpeg.Format) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	m := mime.TypeByExtension(ext)
+	m := ext2mime[ext]
```

Comment:
`if m, ok := ext2mime[ext]; ok { return m, nil }`

Reply:
moved condition to be one the same line, but not used `ok` flag - don't see any need for that

Reply:
Zero values can be semantically awkward ("does this key actually exist but is empty?") so rather than having future developers check whether this is always safe for a given map/table, I'd rather we just be consistent throughout the codebase and use the "comma-ok" idiom.

Not that I expect we'll ever actually have an extension mapping to an empty mime type, but the consistency feels important from a code quality / robustness / readability perspective.

Reply:
> "does this key actually exist but is empty?"

It this particular place it doesn't matter - empty mime type is error here.
But I've also added check for key existence.